test/uk/gov/hmrc/economiccrimelevyregistration/controllers/StartControllerSpec.scala

```diff
@@ -53,26 +44,12 @@ class StartControllerSpec extends SpecBase {
   }
 
   "onSubmit" should {
-    "redirect to register for current year page if no return url" in {
-      when(mockSessionService.getOptional(any(), any(), ArgumentMatchers.eq(SessionKeys.UrlToReturnTo))(any()))
-        .thenReturn(EitherT.fromEither[Future](Right(None)))
-
+    "redirect to AML Activity page if no return url" in {
```

Comment:
The name of what you are redirecting to needs changing.

Reply:
Fixed.